packages/code-connect/components/Alert/InlineAlertGroup.figma.tsx

```diff
@@ -0,0 +1,18 @@
+import figma from '@figma/code-connect';
+import { AlertGroup } from '@patternfly/react-core';
+
+/**
+ * PatternFly Alert component integration for Figma Code Connect
+ */
+
+figma.connect(
+  AlertGroup,
+  'https://www.figma.com/design/aEBBvq0J3EPXxHvv6WgDx9/PatternFly-6--Components-Test?node-id=1110-2780&m=dev',
+  {
+    props: {
+      // children
+      children: figma.children('*')
+    },
+    example: (props) => <AlertGroup>{props.children}</AlertGroup>
```

Comment:
These AlertGroups should have `isLiveRegion` and `hasAnimations` if they could appear on the page. They don't need these props if they are present on page load and cannot be dismissed.